packages/webpack-config/src/index.js

```diff
@@ -102,25 +70,70 @@ const config = {
 				parser: { requireEnsure: false },
 			},
 
-			// Stylesheets - if the extension is .js.scss, leave the CSS embedded in the JS file
+			// CSS loaders, which should apply to all CSS, including SCSS
 			{
-				test: /\.js\.scss$/,
-				use: ['style-loader', ...cssLoaders],
+				test: /(\.css)|(\.scss)$/,
+				use: [
+					// css-loader resolves paths in CSS and adds assets as dependencies.
+					{
+						loader: 'css-loader',
+						options: {
+							sourceMap: (mode === 'development'),
+							minimize: (mode === 'production'),
+							url: false,
+						},
+					},
+					// postcss-loader automatically applies browser prefixes to our css.
+					{
+						loader: 'postcss-loader',
+						options: {
+							sourceMap: (mode === 'development'),
+							plugins: () => [
+								autoprefixer,
+							],
+						},
+					},
+				],
 			},
 
-			// Stylesheets - if the extension is .scss, extract the CSS into its own file
+			// sass-loader converts our sass to css
 			{
 				test: /\.scss$/,
-				use: [MiniCssExtractPlugin.loader, ...cssLoaders],
+				use: [
+					{
+						loader: 'sass-loader',
+						options: {
+							sourceMap: (mode === 'development'),
+							// Set scss debug flag
+							data: `$IS_DEBUG: ${(mode === 'development')};`,
+						},
+					},
+				],
+			},
+
+			// Custom CSS post loaders which apply conditionally
+			// If a stylesheet is imported into JavaScript, leave the CSS embedded in the JS and dynamically inject into the web page
+			// Otherwise, extract the CSS into its own file
+			{
+				test: /(\.css)|(\.scss)$/,
+				enforce: 'post', // These loaders need to be run after the other CSS loaders
+				oneOf: [
+					{
+						issuer: /\.js$/,
```

Comment:
Oh no, this doesn't work :(
`issuer` only looks one level up the module tree. If this CSS file is imported into another CSS file, which is imported into a JS file, then it is going to be extracted, when it should remain embedded in the JS.

Reply:
I think this is acceptable actually. I'll just need to document the behaviour.

Reply:
Documentation updated